tests/it/execution.rs

```diff
@@ -343,3 +343,149 @@ fn can_execute_for_in_empty_list() {
         "#},
     );
 }
+
+#[test]
+fn can_execute_star_capture_non_empty() {
+    check_execution(
+        indoc! {r#"
+          pass
+          pass
+        "#},
+        indoc! {r#"
+          (module (_)* @stmts)
+          {
+            var n = 0
+            for x in @stmts {
+              set n = (plus n 1)
+            }
+            node node0
+            attr (node0) val = n
+          }
+        "#},
+        indoc! {r#"
+          node 0
+            val: 2
+        "#},
+    );
+}
+
+#[test]
+fn can_execute_star_capture_empty() {
+    check_execution(
+        indoc! {r#"
+        "#},
+        indoc! {r#"
+          (module (_)* @stmts)
+          {
+            var n = 0
+            for x in @stmts {
+              set n = (plus n 1)
+            }
+            node node0
+            attr (node0) val = n
+          }
+        "#},
+        indoc! {r#"
+          node 0
+            val: 0
+        "#},
+    );
+}
+
+#[test]
+fn can_execute_plus_capture_non_empty() {
+    check_execution(
+        indoc! {r#"
+          pass
+          pass
+        "#},
+        indoc! {r#"
+          (module (_)+ @stmts)
+          {
+            var n = 0
+            for x in @stmts {
+              set n = (plus n 1)
+            }
+            node node0
+            attr (node0) val = n
+          }
+        "#},
+        indoc! {r#"
+          node 0
+            val: 2
+        "#},
+    );
+}
+
+#[test]
+fn can_execute_plus_capture_empty() {
```

Comment:
I mean, is this _really_ showing it can execute a `+` when the capture is empty? :slightly_smiling_face: 

It looks more like it's `does_not_execute_plus_capture_empty`.

Reply:
Fair point. In that case I'll drop this test, as it really just tests for `tree-sitter` behavior then.